webapp/src/components/preview-movies/style/recommendation/Recommendation.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+function Recommendation({ average } : { average: number }) {
+  average = Math.round((average * 1000) / 100);
```

Comment:
on ne modifie jamais une props !

```suggestion
  const averageGrade = formatRecommendation(average);
```